pretext/Lists/StringsandLists.ptx

```diff
@@ -40,6 +52,51 @@ print("".join(wds))
   </program>
   <p>The list that you glue together (<c>wds</c> in this example) is not modified.  Also,
             you can use empty glue or multi-character strings as glue.</p>
+
+  <p>
+    <term>index Method in Lists and Strings</term>
+  </p>
+<p> One important thing to note is that even though index methods work the same in the context of strings and lists, 
+  however, they look for different things. In the context of strings, 
+  index methods look for a substring (or a character), whereas in the context of lists, index methods look for an element. 
```

Comment:
One important thing to note is that even though the `index` method is named the same for both strings and lists, they behave differently. 
With strings, the index method looks for a substring (or a character) and returns the first instance of that character.
However, with lists, the index method looks for an element and returns the first instance of that element.

Reply:
changed!